### PR TITLE
feat: customizable imagePullSecrets

### PR DESCRIPTION
**Pull request by Senorsen**
Replace hardcoded imagePullSecrets "registry-credentials" to values 😄

### CODE REVIEW

File: charts/trino/values.yaml [hunk 1]
```diff
@@ -8,6 +8,11 @@ image:
   # Overrides the image tag whose default is the chart version.
   tag: latest
 
+imagePullSecrets: []
+# For example:
+# imagePullSecrets:
+#   - name: registry-credentials
```
Comment (hashhar) [inline charts/trino/values.yaml:14]:
```suggestion
#   - name: pull-secret
```

to make it clear that it doesn't have to be `registry-credentials`.

**Review by hashhar — APPROVED** [review]

**Comment by michael-datareply** [comment]
This would be really great, can we somehow get this merged? :)

**Comment by dnskr** [comment]
@Senorsen Could you please sign CLA to be able to merge our PR?

**Comment by Senorsen** [comment]
@dnskr @cla-bot CLA already signed.

**Comment by hashhar** [comment]
@martint can you help with CLA here?

**Comment by Senorsen** [comment]
@hashhar @dnskr CLA issue seems to be fixed after a re-push

File: charts/trino/values.yaml [hunk 2]
```diff
@@ -8,6 +8,11 @@ image:
   # Overrides the image tag whose default is the chart version.
   tag: latest
 
+imagePullSecrets: []
```
Comment (hashhar) [inline charts/trino/values.yaml:11]: We should keep the old default to avoid breaking users who upgrade.

Comment (Senorsen) [reply charts/trino/values.yaml:11]: Oh, you are right, I forgot to keep the old value as default.

Comment (Senorsen) [reply charts/trino/values.yaml]: Fixed

**Review by hashhar — APPROVED** [review]

**Pull request merged**